### PR TITLE
Code updates

### PR DESCRIPTION
**Pull request by RickyRapp**
These are the model files where the schema for categories, restaurants, and bookings are stored.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+testing123
```
Comment (yossishalem) [inline test.txt]: what is this ?

File: models/Categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const SchemaCategory = new Schema({
+    catName: {
```
Comment (yossishalem) [inline models/Categories.js:4]: minor but important: use categoryName and not catName

**Review by yossishalem — COMMENTED** [review]
Nice job!

File: test.txt (hunk 1 above)
Comment (RickyRapp) [reply test.txt]: Nothing. Will delete.

**Pull request merged**